docs/changelog.rst

```diff
@@ -39,8 +39,8 @@ Changelog
     * ``https://upload.pypi.org/``
     * ``https://upload.pypi.io/``
 
-    This helps people with ``https://upload.pypi.io`` still in their .pypirc
-    file.
+    This helps people with ``https://upload.pypi.io`` still in their
+    :file:`.pypirc` file.
```

Comment:
Having tried this in the past, doesn't it look for the file locally when doing this? I thought this broke sphinx builds

Reply:
I wonder how long ago you tried it? Using Sphinx 1.7.0 this builds just fine locally and the output looks the same as the backtick syntax would, but I figure it gives us more flexibility in case we someday want to choose a different RtD/Sphinx theme, or use [this cool `{variable}` functionality](http://www.sphinx-doc.org/en/stable/markup/inline.html#role-file).